demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -1757,11 +1800,69 @@ export class DemoMeetingApp implements
       return DefaultDeviceController.synthesizeVideoDevice('smpte');
     }
 
-    if (value === 'None') {
-      return null;
+    return value;
+  }
+
+  private videoFilterToProcessor(videoFilter: string): VideoFrameProcessor | null {
+    this.log(`Choosing video filter ${this.videoFilter}`);
+
+    if (videoFilter === 'Emojify') {
+      return new EmojifyVideoFrameProcessor('🚀');
     }
 
-    return value;
+    if (videoFilter === 'CicurlarCut') {
+      return new CircularCut();
+    }
+
+    if (videoFilter === 'NoOp') {
+      return new NoOpVideoFrameProcessor();
+    }
+
+    return null;
+  }
+
+  private chosenProcessor: { name: string | undefined; processor: VideoFrameProcessor | undefined; } = {
+    name: undefined,
+    processor: undefined
+  };
```

Comment:
You never use `processor` — it's adopted by the transform device. Just track the name: `selectedVideoFilter`.